tools/build_pytorch_libs.py

```diff
@@ -33,8 +32,12 @@ def _create_build_env():
     # you should NEVER add something to this list. It is bad practice to
     # have cmake read the environment
     my_env = os.environ.copy()
-    if USE_CUDA:
-        my_env['CUDA_BIN_PATH'] = CUDA_HOME
+    if 'CUDA_HOME' in my_env:  # Keep CUDA_HOME. This env variable is still used in other part.
```

Comment:
For Windows the variable is called `CUDA_PATH`.

Reply:
I didn't include `CUDA_PATH` here because it has already been handled in `FindCUDA.cmake`: https://github.com/pytorch/pytorch/blob/ee21142e40f15596a1e83c20db79ed6e20c59c93/cmake/Modules_CUDA_fix/upstream/FindCUDA.cmake#L680

Reply:
Thanks for the explanation. That sounds reasonable.